src/bootstrap-fork.js

```diff
@@ -37,9 +37,6 @@ if (process.env['VSCODE_PARENT_PID']) {
 	terminateWhenParentTerminates();
 }
 
-// Configure Crash Reporter
```

Comment:
Did the crash reporter API change, or are we using another newer one now?

Reply:
Yes, Linux will start using crashpad instead of breakpad starting with this version, hence we don't need to start the agent in every child process.